src/UnitOfWork/OperationConsolidationMode.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types = 1);
+
+namespace BrandEmbassy\UnitOfWork;
+
+/**
+ * @final
+ */
+class OperationConsolidationMode
```

Comment:
[O] You cannot consolidate one single Operation, so wouldn't be better plural?
```suggestion
class OperationsConsolidationMode
```

Reply:
I follow already set naming "OperationConsolidator"